### DISCUSSION TITLE
How to overlay controls on top of each other? (to create a big `ProgressRing` with `Text` at its center)

### DISCUSSION THREAD

**Discussion by quan787** [flet-dev/flet#341] [Ideas]
![image](https://user-images.githubusercontent.com/16741020/190852874-84f7409d-b5c8-4df5-b672-a6fab8720565.png)

It'll be great to have `ProgressRing` with text property, too

**Comment by FeodorFitsner** [comment]
Here you are: https://github.com/flet-dev/examples/blob/main/python/controls/progress-ring/gauge-with-progress.py

Use `Stack`.

**Reply by quan787** [reply]
Much thanks! I totally missed this part in the doc and now this question looks obvious and stupid :P
